parent/pom.xml

```diff
@@ -248,6 +243,18 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>deploy</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.sonatype.plugins</groupId>
+                        <artifactId>nexus-staging-maven-plugin</artifactId>
```

Comment:
Heads up that this plugin has [historically had trouble with Java 17](https://github.com/sonatype/nexus-public/issues/110#issuecomment-1547272513)